community/how_to_ask_on_discourse.md

```diff
@@ -1,4 +1,128 @@
 (how_to_ask_on_discourse)=
-# How to ask a question on Discourse
+# Asking Questions on the PyMC Discourse
+
+This document provides some advice for those asking questions on the [PyMC Discourse site](https://discourse.pymc.io/), the main channel for communication among the PyMC community.  If you have questions regarding modeling, PyMC/Aesara/arviz usage, or Bayesian modeling in general, please create a new post and someone is likely help you out.
+
+The items in this guide are designed to help maximize the chances that you good answers quickly.
+
+- {ref}`search`
+- {ref}`problem`
+- {ref}`mwe`
+- {ref}`title`
+- {ref}`version`
+- {ref}`tags`
+- {ref}`example`
+
+(search)=
+## Search
+
+The PyMC Discourse is full of questions that have already received useful answers.  So definitely use the search function to see if anyone has asked questions that are similar to yours.  Try different combinations of keywords because past questions don't necessarily include a great {ref}`title`.  If you find similar questions, but the answers do not solve your problem, include links to these past questions in your topic and explain why the answers do not solve your problem.  You can also search for topics by selecting relevant {ref}`tags` (or filter search results by tag).
```

Comment:
I think here we should use custom text for the title reference instead of having sphinx fill it with the actual title as it breaks the reading flow. The section title being referenced is "A good title" so it now reads `a great A good title.`

```suggestion
The PyMC Discourse is full of questions that have already received useful answers.  So definitely use the search function to see if anyone has asked questions that are similar to yours.  Try different combinations of keywords because past questions don't necessarily include a great {ref}`title <title>`.  If you find similar questions, but the answers do not solve your problem, include links to these past questions in your topic and explain why the answers do not solve your problem.  You can also search for topics by selecting relevant {ref}`tags` (or filter search results by tag).
```